.github/workflows/ci_update_requirements.yml

```diff
@@ -0,0 +1,83 @@
+name: CI - Check for updates to `requirements.txt`
+
+on:
+  schedule:
+    # At 8:30 every Monday (6:30 UTC)
```

Comment:
```suggestion
    # At 7:30 every Monday (6:30 UTC)
```

Reply:
I think this depends on summer time or not, right? ;) But I'll add it in.